### PR TITLE
HTML and CSS Popup

### PR DESCRIPTION
**Pull request by MishaDerkach**
### task HTML & CSS Popup

[Demo](https://MishaDerkach.github.io/Popup) |
[Code base](https://github.com/MishaDerkach/Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/MishaDerkach/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="header">
+        <nav class="nav">
```
Comment (A-Ostrovnyy) [inline submissions/MishaDerkach/html-css-popup/index.html:13]: I think you have enough space to put it in one line on mobile)
image

File: submissions/MishaDerkach/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                    <input type="checkbox" id="popup-button" class="popup-input">
+                    <label for="popup-button" class="popup-button">
+                        <img src="popup-icons/popup-button.png" alt="icon" class="popup-icon">
+                    </label>
+
+                    <div class="content">
+                        <ul class="content-list">
+                            <li class="content-list__item">
+                                <a href="#" class="content-list__link">
+                                    <img src="popup-icons/apps.png" alt="Apps" class="content-list__image">
```
Comment (A-Ostrovnyy) [inline submissions/MishaDerkach/html-css-popup/index.html:31]: Read how and when to populate alt attribute value [Как правильно написать alt-текст](https://htmlacademy.ru/blog/articles/alt-text)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Sorry for the late response.

**Comment by MishaDerkach** [comment]
Good evening! I'm so sorry for my late changes in the work. I can't do it earlier. So now modified my popup using your recomendations.

File: submissions/MishaDerkach/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                            </li>
+                            <li class="content-list__item">
+                                <a href="#" class="content-list__link">
+                                    <img src="popup-icons/blogs.png" class="content-list__image">
+                                    <span class="content-list__title">Blogs</span>
+                                </a>
+                            </li>
+                            <li class="content-list__item">
+                                <a href="#" class="content-list__link">
+                                    <img src="popup-icons/calendar.png" class="content-list__image">
```
Comment (A-Ostrovnyy) [inline submissions/MishaDerkach/html-css-popup/index.html:43]:
```suggestion
    <img src="popup-icons/calendar.png" class="content-list__image" alt="">
```
`alt` attribute is required `img` attribute

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Don't worry about that.
Good progress!

**Comment by MishaDerkach** [comment]
Good evening. I modified my index.html by your recomendation.

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Comment by MishaDerkach** [comment]
Thanks!

**Pull request merged**